pages.ja/windows/clip.md

```diff
@@ -0,0 +1,20 @@
+# clip
+
+> 入力コンテンツをWindowsクリップボードにコピーします
+> 詳しくはこちら: <https://docs.microsoft.com/windows-server/administration/windows-commands/clip>.
+
+- コマンドライン出力をWindowsクリップボードにパイプします:
+
+`{{dir}} | clip`
+
+- ファイルの内容をWindowsクリップボードにコピーします:
+
+`clip < {{path/to/file.ext}}`
+
+- 末尾に改行が付いたテキストをWindowsクリップボードにコピーします:
+
+`echo {{some text}} | clip`
```

Comment:
You can translate the arguments like `some text` to be more clear for non-English speakers.

Reply:
I have gone through some of the files again to review what argument can be translated and tried my best, but for some of them like "path/to/directory" I have translated it to "directory path" in Japanese because it doesn't really make much sense in Japanese to literally translate the original text(at least for me).

Can you check the updated files again?

Reply:
Yeah, we understand. If we want a `word by word` translation we would use gTranslate lol